src/Entity/CaptchaPoint.php

```diff
@@ -21,17 +22,25 @@
  *     "form" = {
  *       "add" = "Drupal\captcha\Form\CaptchaPointForm",
  *       "edit" = "Drupal\captcha\Form\CaptchaPointForm",
+ *       "disable" = "Drupal\captcha\Form\CaptchaPointDisableForm",
+ *       "enable" = "Drupal\captcha\Form\CaptchaPointEnableForm",
  *       "delete" = "Drupal\captcha\Form\CaptchaPointDeleteForm"
  *     }
  *   },
  *   config_prefix = "captcha_point",
  *   admin_permission = "administer CAPTCHA settings",
+ *   list_cache_tags = {
+ *    "rendered"
```

Comment:
We need to document somewhere why we do this. There can't be comments inside the annotation, so it will have to be above. Explain that adding or removing a captcha could affect any forms, that might be displayed anywhere, so changes need to invalidate the render cache completely.